onlineweb4/settings/base.py

```diff
@@ -1,28 +1,39 @@
 # -*- coding: utf8 -*-
 import os
 import sys
-from webpack_resolve import create_resolve_file
 
+import dj_database_url
+from decouple import config
 from django.contrib.messages import constants as messages
+from webpack_resolve import create_resolve_file
 
 # Directory that contains this file.
 PROJECT_SETTINGS_DIRECTORY = os.path.dirname(globals()['__file__'])
 # Root directory. Contains manage.py
 PROJECT_ROOT_DIRECTORY = os.path.join(PROJECT_SETTINGS_DIRECTORY, '..', '..')
 
-TEST_RUNNER = "django_nose.NoseTestSuiteRunner"
+sys.dont_write_bytecode = config("OW4_PYTHON_DONT_WRITE_BYTECODE", cast=bool, default=True)
```

Comment:
You really only need to disable it if you don't have write permissions to the directory where the interpereter would write it, or use another featue of the bytecode config to write it somewhere else if it is being run from a network disk for instance.

The interpereter will generate new files if the `.py` file is newer than the bytecode file anyway.
It will help time to first byte after restarting the process, since every import does not have to be re-compiled. Not a big deal, but it is noticable when you restart the ow4 process.